### ISSUE TITLE
unarchive remote_src documentation contradicts itself

### ISSUE THREAD

**Issue by ansibot** [ansible/ansible#29587]
From @ex-nerd on 2016-10-13T00:58:46Z
##### ISSUE TYPE
- Documentation Report
##### COMPONENT NAME

unarchive
##### ANSIBLE VERSION

```
ansible 2.1.2.0
  config file = ***redacted***
  configured module search path = Default w/o overrides
```
##### CONFIGURATION

n/a
##### OS / ENVIRONMENT

n/a
##### SUMMARY

First, the documentation for unarchive, available at [here](http://docs.ansible.com/ansible/unarchive_module.html), describes the `remote_src` parameter as:

```
remote_src
(added in 2.2)
```

Later, under the `src` parameter description, it says:

```
If remote_src=yes and src contains ://, the remote machine will download the file from the url first. (version_added 2.0)
```

With an example provided as:

``` yaml
# Unarchive a file that needs to be downloaded (added in 2.0)
- unarchive: src=https://example.com/example.zip dest=/usr/local/bin remote_src=yes
```

If I look at the same example documentation from my local copy of `unarchive.py` (version `2.1.2.0`), it says:

``` yaml
# Unarchive a file that needs to be downloaded (added in 2.0)
- unarchive: src=https://example.com/example.zip dest=/usr/local/bin copy=no
```

This seems to suggest that the `(added in 2.0)` comments refer to the ability to use a remote source, but the actual behavior changed in `2.2` with the addition of the `remote_src` parameter.   The documentation should be updated to reflect the different behavior required for working with remote source files between `2.0` (`copy=no` and no `remote_src`) and `2.2` (`remote_src=yes` and no `copy` parameter), or at the very least the "added in 2.0" strings should be removed as they now imply that `remote_src` was added in 2.0, which it was not.

Copied from original issue: ansible/ansible-modules-core#5246

**Comment by ansibot** [comment]
From @ansibot on 2016-10-13T00:58:46Z



@dagwieers, @pileofrogs, ping. This issue is waiting on your response.
[click here for bot help](https://github.com/ansible/ansibullbot/blob/master/ISSUE_HELP.md)

**Comment by ansibot** [comment]
From @dagwieers on 2016-10-13T00:58:46Z

Right, I don't feel compelled to do anything for this, as I was not involved with the change to `remote_src` and I don't agree with this change. I think it was a bad choice to change it as was done as it only adds to the confusion.

Secondly, the "added in 2.0" reflects the downloading from URLs, not the change from `copy` to `remote_src` AFAIU. needs_contributor

**Comment by ansibot** [comment]
From @ex-nerd on 2016-10-13T00:58:46Z

Seems like a simple change to the example's comment would clear things up.  Something like  `added in 2.0, remote_src behavior changed in 2.2`

**Comment by biomassives** [comment]
This message was created automatically by mail delivery software.

A message that you sent could not be delivered to one or more of its
recipients. This is a temporary error. The following address(es) deferred:

  acmeideal@gmail.com
    Domain biomassiv.es has exceeded the max emails per hour (113/100 (112%)) allowed.  Message will be reattempted later

------- This is a copy of the message, including all the headers. ------
------ The body of the message is 6667 characters long; only the first
------ 5000 or so are included here.
Received: from github-smtp2-ext1.iad.github.net ([192.30.252.192]:57000 helo=github-smtp2a-ext-cp1-prd.iad.github.net)
	by chi-server32.websitehostserver.net with esmtps (TLSv1.2:ECDHE-RSA-AES256-GCM-SHA384:256)
	(Exim 4.89)
	(envelope-from <noreply@github.com>)
	id 1drPyc-000m7Y-MO
	for greg@biomassiv.es; Mon, 11 Sep 2017 09:46:18 -0500
Date: Mon, 11 Sep 2017 07:45:17 -0700
DKIM-Signature: v=1; a=rsa-sha256; c=relaxed/relaxed; d=github.com;
	s=pf2014; t=1505141117;
	bh=/ZOsOg2N8Obx2mS23c3DXe/Yl0Nh/rsrNdu5dfNbFqk=;
	h=From:Reply-To:To:Cc:Subject:List-ID:List-Archive:List-Post:
	 List-Unsubscribe:From;
	b=Sy5pFIjNPf1sQMmPC7FU2SfKqIxl1py6YPCu5pcKclhBNuWkEg+2BDLd9yXmpsjgi
	 T2Z6PfChWry2LDfIJC3GS1jHIhlzYWFnPZ7gScvGW/GW2T/yyRWGwhtB6bTcUGdSyx
	 ALppD4+omP/vsSv+hX66afrpeuxknCD/RXBRJOTg=

**Comment by sijis** [comment]
It seems the current documentation has updated the content and seems to reflect things properly.
* Parameter `remote_src` says (added in 2.2)
* The `copy` param also now notes:
`This option has been deprecated in favor of remote_src.
This option is mutually exclusive with remote_src.`
